packages/patterns-react/src/patterns/LeadSpace/__stories__/LeadSpace.stories.js

```diff
@@ -14,5 +14,40 @@ storiesOf('LeadSpace', module)
     },
   })
   .add('Default', () => {
-    return <LeadSpace />;
+    const copy = text(
+      'copy',
+      'Use this area for a short line of copy to support the title'
+    );
+
+    const title = text('title', 'Lead space title');
+
+    const variations = {
+      expressive: '',
+      productive: 'productive',
+    };
+
+    const imageUrl = text('imageUrl', 'https://picsum.photos/id/1025/1000/400');
+
+    const buttons = [
+      {
+        link: '',
```

Comment:
Have you tested this by filling this out yet? Apparently it changes to an `a` tag when you add this property, and it changes the design up.

![image](https://user-images.githubusercontent.com/3793636/64735724-c241c900-d4ae-11e9-8b03-633a1bb7706e.png)